source/guide_gitea.rst

```diff
@@ -96,93 +157,276 @@ and a new config file at ``~/gitea/custom/conf/app.ini`` to specify the port and
 
- [isabell@stardust ~]$ ln -s ~/.ssh ~/gitea/
+.. code-block:: config
 
-Add your key in gitea and then change the line in ``.ssh/authorized_keys`` to the following:
+  command="/home/isabell/gitea/gitea --config='/home/isabell/gitea/custom/conf/app.ini' serv key-1",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-ed25519 AAAAC... youruser@yourhost
 
-.. code-block:: sh
+If we're using the same ssh key for auth to uberspace and Gitea, we need to modify the server ``/home/isabell/.ssh/authorized_keys`` file.
 
- command="if [ -t 0 ]; then bash; elif [[ $SSH_ORIGINAL_COMMAND =~ ^(scp|rsync|mysqldump).* ]]; then eval $SSH_ORIGINAL_COMMAND; else /home/<username>/gitea/gitea serv key-1 --config='/home/<username>/gitea/custom/conf/app.ini'; fi",no-port-forwarding,no-X11-forwarding,no-agent-forwarding ssh-...
+.. code-block:: config
 
-.. warning:: Replace ``<username>`` with your username and put your public key after ``ssh-``!
+  command="if [ -t 0 ]; then bash; elif [[ ${SSH_ORIGINAL_COMMAND} =~ ^(scp|rsync|mysqldump).* ]]; then eval ${SSH_ORIGINAL_COMMAND}; else /home/isabell/gitea/gitea serv key-1 --config='/home/isabell/gitea/custom/conf/app.ini'; fi",no-port-forwarding,no-X11-forwarding,no-agent-forwarding ssh-...
 
-Tuning
-======
+.. note:: Be careful to keep the key number ``key-X``, keep your key ``ssh-...`` and change the username isabell to yours.
 
-You can easily add AsciiDoc rendering to Gitea. Install asciidoctor via ruby gems:
+.. note:: You can still use the Uberspace dashboard_ to add ssh keys.
 
-::
+To interact with Gitea at our local machine like ``git clone isabell@isabell.uber.space:giteauser/somerepo.git`` we configure the ``/home/localuser/.ssh/config`` file for our local machine with the git ssh key.
 
- [isabell@stardust ~]$ gem install asciidoctor
+.. code-block:: config
 
-Edit your config file at ``~/gitea/custom/conf/app.ini`` to enable asciidoctor in Gitea:
+  Host isabell.uber.space
+      HostName isabell.uber.space
+      User isabell
+      IdentityFile ~/.ssh/id_your_git_key
+      IdentitiesOnly yes
+
+
+Finishing installation
+======================
+
+Uberspace daemon for Gitea
+---------------------------
+
+Create a file ``~/etc/services.d/gitea.ini`` for the service ...
 
 .. code-block:: ini
 
-  [markup.asciidoc]
-  ENABLED = true
-  FILE_EXTENSIONS = .adoc,.asciidoc
-  RENDER_COMMAND = "asciidoctor -e -a leveloffset=-1 --out-file=- -"
-  IS_INPUT_FILE = false
+  [program:gitea]
+  command=%(ENV_HOME)s/gitea/gitea web
+
+.. include:: includes/supervisord.rst
+
+.. note:: The status of gitea must be ``RUNNING``. If not, check the configuration file ``~/gitea/custom/conf/app.ini``.
+
+Uberspace web backend
+---------------------
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ uberspace web backend set / --http --port 9000
```

Comment:
Please us the web backend snippet here:

```
.. note::

    gittea is running on port 9000.

.. include:: includes/web-backend.rst
```

I agree that it leads to a less pretty result, but makes the guide much more manageable for us.

Reply:
Done.